### PR TITLE
SDIT-2044 A few more fields to PERSON ADDRESS

### PR DESCRIPTION
**Pull request by andymarke**
City, County, Country

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/Address.kt [hunk 1]
```diff
@@ -66,26 +66,8 @@ abstract class Address(
   @Column(name = "POSTAL_CODE")
   open val postalCode: String? = null,
 
-) {
```
Comment (andymarke) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/Address.kt:69]: city, county, country moved up to constructor - PR diff just looks like the opposite has happened

File: src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/Address.kt [hunk 2]
```diff
@@ -97,10 +79,9 @@ abstract class Address(
       ), JoinColumnOrFormula(column = JoinColumn(name = "COUNTY_CODE", referencedColumnName = "code")),
     ],
   )
-  open val county: County? = null
+  open val county: County? = null,
 
   @ManyToOne
-  @NotFound(action = NotFoundAction.IGNORE)
```
Comment (andymarke) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/Address.kt:103]: data all correct in prod so removed

File: src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/Address.kt [hunk 3]
```diff
@@ -111,10 +92,9 @@ abstract class Address(
       ), JoinColumnOrFormula(column = JoinColumn(name = "CITY_CODE", referencedColumnName = "code")),
     ],
   )
-  open val city: City? = null
+  open val city: City? = null,
 
   @ManyToOne
-  @NotFound(action = NotFoundAction.IGNORE)
```
Comment (andymarke) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/Address.kt:117]: data all correct in prod so removed

File: src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/PersonAddress.kt [hunk 4]
```diff
@@ -25,26 +32,32 @@ interface PersonAddressDsl {
 @Component
 class PersonAddressBuilderFactory(
   private val addressPhoneBuilderFactory: AddressPhoneBuilderFactory,
-  private val addressPhoneBuilderRepository: PersonAddressBuilderRepository,
+  private val personAddressBuilderRepository: PersonAddressBuilderRepository,
```
Comment (andymarke) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/PersonAddress.kt:35]: corrected variable name

**Review by mikehalmamoj — APPROVED** [review]

**Pull request merged**